ThunderCloud/CollectionCell.swift

```diff
@@ -57,8 +60,8 @@ open class CollectionCell: StormTableViewCell {
 		
 		sharedInit()
         
-        let cellNib = UINib(nibName: "CollectionItemViewCell", bundle: Bundle(for: CollectionCell.self))
-        collectionView.register(cellNib, forCellWithReuseIdentifier: "Cell")
+        let cellNib = UINib(nibName: CollectionCell.CollectionItemViewCellNibName, bundle: Bundle(for: CollectionCell.self))
+        collectionView.register(cellNib, forCellWithReuseIdentifier: CollectionCell.CollectionItemViewCellNibName)
```

Comment:
Is the view file also updated?

Reply:
How do you mean? :)

Reply:
Has the cell reuse indentifier in IB also been changed to be the same as the nibName? :D

Reply:
Ahh gotcha! `CollectionCell` doesn't use `IB` currently so no need! Good spot though to check!